<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>627</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>5</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-color: rgb(0, 67, 49);
background-color: rgb(180, 255, 226);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="win_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>511</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dosis ExtraBold</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;
color: rgb(182, 182, 182);
background-color: rgb(0, 63, 46);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="back_hand_btn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>490</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <strikeout>false</strikeout>
      <stylestrategy>PreferAntialias</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 39px;
color: rgb(0, 145, 0);
background-color: rgb(0, 67, 49);</string>
    </property>
    <property name="text">
     <string>ü§öüèª</string>
    </property>
   </widget>
   <widget class="QPushButton" name="front_hand_btn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>490</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 39px;
color: rgb(0, 145, 0);
background-color: rgb(0, 67, 49);</string>
    </property>
    <property name="text">
     <string>‚úãüèª</string>
    </property>
   </widget>
   <widget class="QPushButton" name="player_score_btn">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>440</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dosis ExtraBold</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;
color: rgb(50, 50, 50);</string>
    </property>
    <property name="text">
     <string>Your Score</string>
    </property>
   </widget>
   <widget class="QPushButton" name="computer1_score">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dosis ExtraBold</family>
      <pointsize>15</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;
color: rgb(50, 50, 50);</string>
    </property>
    <property name="text">
     <string>Computer1 Score </string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>171</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border :2px ;
border-color: rgb(170, 0, 127);
border-style : dashed;
border-radius: 38px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="computer1_choice">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>171</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>43</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border :2px solid purple;
border-style : dashed;
border-radius: 38px;
border-color: rgb(0, 67, 49);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>79</y>
      <width>171</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border :2px ;
border-color: rgb(170, 0, 127);
border-style : dashed;
border-radius: 38px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="computer2_choice">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>171</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>43</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border :2px solid purple;
border-style : dashed;
border-radius: 38px;
border-color: rgb(0, 67, 49);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="computer2_score">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dosis ExtraBold</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;
color: rgb(50, 50, 50);</string>
    </property>
    <property name="text">
     <string>Computer2 Score</string>
    </property>
   </widget>
   <widget class="QPushButton" name="player_choice">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>310</y>
      <width>171</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>43</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border :2px solid purple;
border-style : dashed;
border-radius: 38px;
border-color: rgb(0, 67, 49);

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>541</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
